jwst/datamodels/model_base.py

```diff
@@ -368,71 +368,15 @@ def callback(schema, path, combiner, ctx, recurse):
 
         mschema.walk_schema(self._schema, callback, ctx=self)
 
-    def info(self):
-        """
-        Return datatype and dimension for each array or table
-        """
-        def get_field_info(path, instance):
-            field_info = []
-            if isinstance(instance, dict):
-                if path:
-                    path += '.'
-                for name, val in instance.items():
-                    new_path = path + name.lower()
-                    field_info.extend(get_field_info(new_path, val))
-            elif isinstance(instance, list):
-                for index, val in enumerate(instance):
-                    new_path = "%s[%d]" % (path, index)
-                    field_info.extend(get_field_info(new_path, val))
-            elif isinstance(instance, np.ndarray):
-                if instance.shape[0] > 0:
-                    shape_info = get_shape_info(instance)
-                    type_info = get_type_info(instance)
-                    field_info = [(path, shape_info, type_info)]
-            return field_info
-
-        def get_meta_info(meta):
-            meta_info = []
-            for attribute in ('filename', 'date', 'model_type'):
-                if hasattr(meta, attribute):
-                    value = getattr(meta, attribute)
-                    if value is not None:
-                        meta_info.append(attribute + ': ' + value)
-            return meta_info
-
-        def get_shape_info(instance):
-            if hasattr(instance, '_coldefs'):
-                nrows = instance.shape[0]
-                ncols = len(instance._coldefs)
-                shape_info = "{}R x {}C".format(nrows, ncols)
-            else:
-                # Display shape in fits order (reversed)
-                shape = [str(s) for s in reversed(instance.shape)]
-                shape_info = '(' + ','.join(shape) + ')'
-            return shape_info
-
-        def get_type_info(instance):
-            if hasattr(instance, '_coldefs'):
-                col_info = []
-                for coldef in instance._coldefs:
-                    col_info.append(coldef.name + ':' + coldef.format)
-                type_info = '(' + ','.join(col_info) + ')'
-            else:
-                type_info = numpy_dtype_to_asdf_datatype(instance.dtype)[0]
-            return type_info
+    def info(self, **kwargs):
+        return self._asdf.info(**kwargs)
 
-        meta_info = get_meta_info(self.meta)
-        field_info = get_field_info('', self._instance)
+    info.__doc__ = asdf.info.__doc__
```

Comment:
Because these are methods on `self`, the docstrings are wrong in the first arg.  It doesn't exist in this implementation.

Reply:
Sorry, which implementation? This PR should be merged after the release of asdf 2.6. The asdf version requirement will change to >=2.6. Is this what you mean?

Reply:
No, `asdf.info.__doc__` shows

```
    Print a rendering of an ASDF tree or sub-tree to stdout.

    Parameters
    ----------
    node_or_path : str, pathlib.Path, asdf.asdf.AsdfFile, or any                    ASDF tree node
        The tree or sub-tree to render.  Strings and Path objects
        will first be passed to asdf.open(...).

    max_rows : int, tuple, or None, optional
        Maximum number of lines to print.  Nodes that cannot be
        displayed will be elided with a message.
        If int, constrain total number of displayed lines.
        If tuple, constrain lines per node at the depth corresponding             to the tuple index.
        If None, display all lines.
```

But because in `datamodels`, `info` is a method that operates on `self`, `node_or_path` is not part of the calling sequence.  The docstring should not include `node_or_path`.